pareto/other_models/CM_module/operational/screening_tool.ipynb

```diff
@@ -0,0 +1,801 @@
+      "text/plain": [
+       "<Figure size 2700x1500 with 8 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "min_node, models = node_rerun(\n",
+    "    df_sets, df_parameters, treatment_site=\"R01\", max_iterations=5000\n",
+    ")\n",
+    "final_model = models[min_node]\n",
+    "print()\n",
+    "print(f\"*** Lowest cost location for treatment ***\")\n",
+    "\n",
+    "# Displaying specific broken down costs\n",
+    "print_results_summary(final_model)"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Turns out N03 wasn't the best location to install the desalination facility rather N06 is the optimal installation location. This makes sense because N06 is next to a high lithium concentration production pad."
```

Comment:
```suggestion
    "It turns out that the initial location of the desalination facility next to node N03 is suboptimal. Rather, N06 is the optimal installation location. This makes sense because N06 is next to a high lithium concentration production pad."
```
You're implying that the original location of the desalination facility is N03, but I don't think that's quite true. Maybe your model is a bit different, but in the PARETO toy strategic case study, N03 and R01 are next to each other, but they aren't the same location.